session_commands.go

```diff
@@ -43,6 +43,11 @@ type SessionCommands interface {
 	// it requires the SDR.
 	GetSensorReading(context.Context, uint8) (*ipmi.GetSensorReadingRsp, error)
 
+	// SetSessionPrivilegeLevel sends a Set Session Privilege Level command to the BMC. This is
+	// specified in 18.16 and 22.18 of IPMI v1.5 and 2.0 respectively.
+	// If the PrivilegeLevel argument is 0, it will return the current session privilege level.
```

Comment:
Let's avoid mentioning 0 like we did for the field in the request struct.

Reply:
You are right, but in this case the caller can't omit this argument (not like for the `SetSessionPrivilegeLevelReq struct`).
So I think it is important to document this corner case (however the phrasing can surely be improved).

Reply:
I'm tempted to enhance the spec and create `GetSessionPrivilegeLevel(context.Context) (ipmi.PrivilegeLevel, error)` which simply returns `SetSessionPrivilegeLevel(ctx, PrivilegeLevel(0))`. I think that's neater than creating a new `PrivilegeLevel` const like `PrivilegeLevelCurrent = 0`.